libs/service-portal/access-control/delegations/src/screens/AccessControl/AccessControl.tsx

```diff
@@ -0,0 +1,127 @@
+import { Query } from '@island.is/api/schema'
+import { useAuth } from '@island.is/auth/react'
+
+export const AuthDelegationsQuery = gql`
+  query AuthDelegationsListQuery {
+    authDelegations {
+      ... on AuthCustomDelegation {
+        validTo
+      }
+    }
+  }
+`
+
+const AccessControl: ServicePortalModuleComponent = ({ userInfo, client }) => {
+  useNamespaces('sp.settings-access-control')
+  const { data, loading } = useQuery<Query>(AuthDelegationsQuery)
+  const { switchUser } = useAuth()
+  const { formatMessage } = useLocale()
+
+  if (!loading && data?.authDelegations.length === 0) {
+    return (
+      <NoDataScreen
+        title={formatMessage({
+          id: 'sp.settings-access-control:empty-title',
+          defaultMessage: 'Umboð',
+        })}
+        button={{
+          type: 'internal',
+          link: ServicePortalPath.AccessControlDelegationsGrant,
+          text: formatMessage({
+            id: 'sp.settings-access-control:empty-new-access',
+            defaultMessage: 'Veita aðgang',
+          }),
+          variant: 'primary',
+        }}
+        secondaryButton={{
+          type: 'click',
+          onClick: () => switchUser(),
+          text: formatMessage({
+            id: 'sp.settings-access-control:empty-switch-access',
+            defaultMessage: 'Skipta um notanda',
+          }),
+          variant: 'ghost',
+        }}
+      >
+        <Text>
+          {formatMessage({
+            id: 'sp.settings-access-control:empty-intro',
+            defaultMessage:
+              'Hérna kemur listi yfir þau umboð sem þú hefur gefið öðrum. Þú getur eytt umboðum eða bætt við nýjum.',
+          })}
+        </Text>
+      </NoDataScreen>
+    )
+  }
+  const actor = userInfo.profile.actor
+  const isDelegation = Boolean(actor)
+
+  const isCompany = userInfo.profile.subjectType === 'legalEntity'
+  const personDelegation = isDelegation && !isCompany
+
+  if (personDelegation) {
+    return <AccessDenied userInfo={userInfo} client={client} />
+  }
+
+  const tabs = [
+    {
+      label: 'Umboð frá mér',
+      content: (
+        <Box marginTop={8}>
+          <IntroHeader
+            title={m.accessControl}
+            intro={formatMessage({
+              id: 'sp.settings-access-control:home-intro',
+              defaultMessage:
+                'Hérna kemur listi yfir þau umboð sem þú hefur gefið öðrum. Þú getur eytt umboðum eða bætt við nýjum.',
+            })}
+          />
+          <Accesses />
+        </Box>
+      ),
+    },
+    {
+      label: 'Umboð til mín',
+      content: (
+        <Box marginTop={8}>
+          <IntroHeader
+            title={m.accessControl}
+            intro={formatMessage({
+              id: 'sp.settings-access-control:home-intro',
+              defaultMessage:
+                'Hérna kemur listi yfir þau umboð sem þú hefur gefið öðrum. Þú getur eytt umboðum eða bætt við nýjum.',
+            })}
+          />
+          <Accesses />
```

Comment:
Isn't this displaying the same thing? Accesses given to others?

Reply:
Yes, this PR is just to get the show running so you can just ignore this.
Modified API endpoints have not yet been created and the design is well on its way. The plan was to basically just start with a copy-paste project and then gradually add updates on screen bases.
I started adding dummy tabs but this will all be updated accordingly in time :)

Reply:
Ah okey I see, thanks for the info :)